spieleplattform-backend/src/main/kotlin/com/example/spieleplattformbackend/security/JwtTokenProvider.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.spieleplattformbackend.security
+
+import io.jsonwebtoken.JwtException
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.security.Keys
+import org.springframework.stereotype.Component
+import java.security.Key
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.*
+
+@Component
+class JwtTokenProvider {
+    private final val secret = "kj32h4jltuvladfjkahrecu19875cn2809437r8"
```

Comment:
Ein secret ist etwas, was über eine Umgebungsvariable in einer Konfigurationsdaei gesetzt werden sollte. Das Secret sollte ja je nach Umgebung (also local, Test, Prod, ...) unterschiedlich sein. Für alle Secrets außer local, würde man diese dann natürlich auch nicht einchecken.